adam-apis/src/main/scala/org/bdgenomics/adam/apis/java/JavaADAMContext.scala

```diff
@@ -31,16 +28,14 @@ import org.bdgenomics.adam.rdd.variant.{
   VariantRDD,
   VariantAnnotationRDD
 }
-import org.bdgenomics.formats.avro._
-import scala.collection.JavaConversions._
 
 object JavaADAMContext {
   // convert to and from java/scala implementations
   implicit def fromADAMContext(ac: ADAMContext): JavaADAMContext = new JavaADAMContext(ac)
   implicit def toADAMContext(jac: JavaADAMContext): ADAMContext = jac.ac
 }
 
-class JavaADAMContext private (val ac: ADAMContext) extends Serializable {
+class JavaADAMContext(val ac: ADAMContext) extends Serializable {
```

Comment:
While we're making this public, we should probably remove the implicits in the companion object, no? They'll be unusable from Javaland.